modules/340-monitoring-kubernetes/templates/node-exporter/daemonset.yaml

```diff
@@ -138,6 +140,10 @@ spec:
       - name: dockersock
         hostPath:
           path: /var/run/docker.sock
+      - name: containerddir
```

Comment:
I assume it would be best to set the `subPath` and abstain from two mounts, or just mount the whole directory.
```yaml
volumes:
- name: containerddir
   hostPath:
     path: /etc/containerd
     type: DirectoryOrCreate
 ...
 volumeMounts:
- name: containerddir
   mountPath: /etc/containerd
   subPath: config.toml
```

Reply:
The whole directory is ok. Fixed